backend/server.js

```diff
@@ -2,14 +2,13 @@ require("dotenv").config();
 const express = require("express");
 const app = express();
 const jwt = require("jsonwebtoken");
-app.use(express.json);
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+const PORT = 3000;
```

Comment:
does this need to be in code?

Reply:
Moved it to the env file and then import it into my server files:
const PORT = process.env.SERVER_PORT;